slices/string/strings.go

```diff
@@ -28,3 +28,18 @@ func Last(arr []string) string {
 
 	return arr[len(arr)-1]
 }
+
+// Unique returns a new slice with only unique values from the original slice.
+func Unique(strSlice []string) []string {
+	keys := make(map[string]bool)
+
+	list := []string{}
```

Comment:
var new_list
for elm in list:
  if elm not in new_list:
     new_list.append(elm)
     
return new_list